### ISSUE TITLE
Filename timestamps increasing at 2x wall clock time on Opus-encoded audio streams

### ISSUE THREAD

**Issue by ijessen-mitll** [bluenviron/mediamtx#4041]
### Which version are you using?

1.10.0

### Which operating system are you using?

Linux amd64 Docker

### Describe how to replicate the issue

1. configure server for recording w/ 1 min segment lengths:

```
  # Record streams to disk.
  record: yes
  # Path of recording segments.
  # Extension is added automatically.
  # Available variables are %path (path name), %Y %m %d %H %M %S %f %s (time in strftime format)
  recordPath: ./recordings/%path/%Y-%m-%d_%H-%M-%S-%f
  # Format of recorded segments.
  # Available formats are "fmp4" (fragmented MP4) and "mpegts" (MPEG-TS).
  recordFormat: fmp4
  # fMP4 segments are concatenation of small MP4 files (parts), each with this duration.
  # MPEG-TS segments are concatenation of 188-bytes packets, flushed to disk with this period.
  # When a system failure occurs, the last part gets lost.
  # Therefore, the part duration is equal to the RPO (recovery point objective).
  recordPartDuration: 1s
  # Minimum duration of each segment.
  recordSegmentDuration: 1m
```

2. start the server
3. publish Opus-encoded audio-only RTSP stream via ffmpeg:

`ffmpeg -i in.wav -f rtsp -c:a libopus rtsp://media-server:8554/streamname`

NOTE:  I've played around with various options, such as `-af aresample=48000` and changing bitrate, bandwidth, etc.  None have had any impact.

4.  observe filenames of recordings.  first file starts at the correct time stamp, all subsequent filenames reflect 2 min durations, despite the actual content (and creation times) reflecting the true 1 minute segment lengths

NOTE: this behavior goes away when I switch the stream encoding to mu-law.

**Comment by ijessen-mitll** [comment]
FYI I'm working around this issue for now with the following configuration to issue commands to correct the filenames:

```
  runOnRecordSegmentCreate: sh -c 'date -u +%Y-%m-%d_%H-%M-%S-%s > $MTX_SEGMENT_PATH.timestamp'
  runOnRecordSegmentComplete: sh -c 'mv $MTX_SEGMENT_PATH $(dirname $MTX_SEGMENT_PATH)/$(head -n 1 $MTX_SEGMENT_PATH.timestamp).mp4 && rm $MTX_SEGMENT_PATH.timestamp'
```

**Comment by aler9** [comment]
Thanks for reporting the issue, this is fixed by #4062
